src/pages/help-centre/components/_utility.ts

```diff
@@ -0,0 +1,50 @@
+import about_deriv from '../data/_about-deriv'
+import account from '../data/_account'
+import affiliate_programme from '../data/_affiliate-programme'
+import dbot from '../data/_dbot'
+import deposits_and_withdrawals from '../data/_deposits-and-withdrawals'
+import deriv_p2p from '../data/_deriv-p2p'
+import derivx from '../data/_deriv-x'
+import dmt5 from '../data/_dmt5'
+import dtrader from '../data/_dtrader'
+import ib_programme from '../data/_ib-programme'
+import multiplier from '../data/_multipliers'
+import security from '../data/_security'
+import trading from '../data/_trading'
+import { TString } from 'types/generics'
+
+export const GENERAL = 'General'
+export const PLATFORMS = 'Platforms'
+export const all_questions = [
```

Comment:
this is not a utility and its better to move somewhere more clear.

Reply:
maybe a constants file

Reply:
You are right, I'll move them somewhere else